npm/private/npm_link_package_store.bzl

```diff
@@ -105,6 +111,11 @@ def _npm_link_package_store_impl(ctx):
         ),
         js_info(
             target = ctx.label,
+            # TODO: if no other JsInfo properties are set, where do they come from?
```

Comment:
All questions I came across while debugging. Can just drop the comments if we want to merge and answer questions/cleanup later.

Reply:
Note that the types from `ctx.attr.src[JsInfo]` are no longer passed through *at all* in this rule.

Is there a missing test for this? Or why would types not have to go through here?

Reply:
Hmm. Guessing we're missing coverage here for a ts_project downstream of a linked JsInfo that requires the types to typecheck.

The changes in npm_package_store.bzl look correct. I think the trick is to add the missing test coverage for types required && populate a full js_info here.

Reply:
It data in:
```
        store_info.files,
        store_js_info.npm_sources,
        store_js_info.sources,
```
Only contain references to the store symlinks such as:
```
<generated file node_modules/.aspect_rules_js/gl-text@1.3.1/node_modules/font-measure>
```

So the package contents (sources, types, random files etc) are no where in here. Those get linked/copied elsewhere?